### PR TITLE
Add a test for the qemu backend

### PR DESCRIPTION
**Pull request by asdil12**

### CODE REVIEW

File: t/data/tests/tests/type_things.pm [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'pitux';
+    send_key 'ret';
+    #type_string "Hallo World";
+    #assert_screen 'hello';
```
Comment (richiejp) [inline t/data/tests/tests/type_things.pm]: Is this a WIP PR?

Comment (asdil12) [inline t/data/tests/tests/type_things.pm]: Yes

File: backend/qemu.pm [hunk 2]
```diff
@@ -675,15 +677,17 @@ sub start_qemu {
...
-        if ($vars->{OFW}) {
-            no warnings 'qw';
-            push(@params, qw(-device nec-usb-xhci -device usb-tablet));
-        }
-        elsif ($vars->{ARCH} eq 'aarch64') {
-            push(@params, qw(-device nec-usb-xhci -device usb-tablet));
-        }
-        else {
-            push(@params, qw(-device usb-ehci -device usb-tablet));
+        unless ($vars->{QEMU_NO_TABLET}) {
```
Comment (coolo) [inline backend/qemu.pm]: add the new variables into docs/variables

**Comment by coolo** [comment]
please add a README.md next to the iso file

**Comment by coolo** [comment]
the crux is that the backend process calls _exit when it's done - and this does not save the cover databas e ;(

**Comment by coolo** [comment]
(see 4af2d4f1c73cf3ef072d9feebacd144de4829ea6 for the reasoning of that)

File: backend/qemu.pm (hunk 2 above)
Comment (asdil12) [inline backend/qemu.pm]: done

**Comment by okurz** [comment]
truly impressive.

**Pull request merged**